traffic_ops/traffic_ops_golang/deliveryservice/keys_test.go

```diff
@@ -2260,6 +2261,7 @@ func TestVerifyAndEncodeCertificateSelfSignedX509v1(t *testing.T) {
 	certChain, certPrivateKey, unknownAuth, _, err := verifyCertKeyPair(SelfSignedX509v1Certificate, SelfSignedX509v1PrivateKey, "", true)
 
 	if err != nil {
+		fmt.Println(err)
 		t.Fatalf("unexpected result: the x509v1 cert/key pair is valid and should have passed validation")
```

Comment:
to print the error that occurred, it should be added to the `t.Fatalf` call instead of printed directly to stdout with `fmt`.

Reply:
Yeah sorry that was my local debug that made it into the commit. It has been removed now.

Reply:
I mean, logging the error was a good idea. Just gotta do it the right way is all

Reply:
Done